src/doc/reference.md

```diff
@@ -2994,7 +2994,7 @@ Some examples of call expressions:
 # fn add(x: i32, y: i32) -> i32 { 0 }
 
 let x: i32 = add(1i32, 2i32);
-let pi: Option<f32> = "3.14".parse().ok();
+let pi: Option<f32> = "3.14".parse();
```

Comment:
I have no compiler to test against, the latest rust-nightly-bin package is (1d00c545e 2015-01-30 19:56:34 +0000). I can't fix this issue and verify that it works until the nightly is updated to include your change; currently I just get:
```
main.rs:12:13: 12:27 error: wrong number of type arguments: expected 1, found 2 [E0244]
main.rs:12     let pi: Option<f32, _> = "3.14".parse();
```